### DISCUSSION TITLE
Using flaps to control speed?!?

### DISCUSSION THREAD

**Discussion by robinottawa** [flybywiresim/aircraft#3987] [Q&A] [answered]
It took a while but I eventually realised that putting on next flaps will slow the target speed on approach. I've now taken to using the flaps to slow the aircraft, but that seems backwards to me. What am I missing? Thx

**Comment by mojoair** [comment, marked as answer]
The speed is adjusted based on your Approach config in MCDU. The managed speed from flaps extraction works only when the Approach phase is active. If you calculate your approach/landing speeds, it will automatically manage the speed to the max allowed at the flap setting down to landing speed at landing config flaps. 

If you lower flaps during you Cruise phase, speed will not reduce and you will get overspeed warnings as your are above the speed threshold for the flap setting.

If you ever notice your speed not reducing as you approach, it is likely because Approach phase did not activate in MCDU (It should automatically activate normally).

I don't think it's backwards but a very useful on approach as it's one less thing to worry about. It's nice that you never have to move your throttle out of CL until you retard for landing/reversers.

**Reply by robinottawa** [reply]
That's great info. Thanks!

I have been learning about the Approach Phase, due to seeing the "enter dest data" message  during arrival. I have learned not to activate it myself (even after I've entered the dest data) as I don't yet know all that it does to the a/c, and since it activates itself at some point (I've started monitoring it). So that's one of the things it does! ;D

**Reply by mojoair** [reply]
That's great. It's a lot to take in especially if you're just getting familiar with more controls. I don't normally plug people's Youtube but there are some excellent tutorials that can be very helpful in understanding these systems but if you get a chance, check out 320simpilot and his tutorials on approach, landing specifically geared towards the A320. He also has many other great tutorials that should cover start-end and all phases in between. 
 
Happy flying.

**Reply by robinottawa** [reply]
I spent quite a bit of time getting up to speed in the a320 by watching 320simpilot. Thanks. It's time I looked at some of his more advanced stuff.

**Reply by Annolago** [reply]
Not to promote one over another, but if you want the best tutorials with explanation then you should look at V1simulation on YouTube. He has a professional series on Airbus going through every part of the flight from planning, take-off, climb, cruise, TOD, approach etc.. He is also have videos on how to do ATC correctly on both vatsim and pilotedge. 
To me the difference is not what he shows, but that he knows the theory behind it and explains it. Maybe this is based on him not being a FO but a Captain and also a certified flight instructor on multiple different aircraft types.. 
But just a recommendation..
